tests/features/bootstrap/FeatureContext.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types = 1);
+
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+
+/**
+ * Defines step definitions that are generally useful in this project.
+ */
+class FeatureContext extends RawDrupalContext {
+}
```

Comment:
This can be removed, with the bootstrap folder. We will add these once we need it.